kanidmd/score/src/https/mod.rs

```diff
@@ -555,7 +553,8 @@ pub fn create_https_server(
         .post(oauth2_id_scopemap_post)
         .delete(oauth2_id_scopemap_delete);
 
-    let mut self_route = appserver.at("/v1/self");
+    // TODO(mea): self -> whoami?
```

Comment:
The "whoami" is just a reflector of details about the currently authenticated user. This is a bit of a "hold-over" from the past, where the auth-bearer tokens used to be opaque, so they couldn't be client-side introspected.

However, recently we have changed these so they CAN be client side introspected, meaning it's possible that we may NOT need the "whoami" route at all.

Reply:
The lines around here:

https://github.com/kanidm/kanidm/blob/master/kanidm_tools/src/cli/common.rs#L115

You could use "uat.uuid" instead which gives you the ID of the token holder, since the tokens are JWT's internally.